### PR TITLE
Disable MSB4011 import warning

### PR DESCRIPTION
**Pull request by pakrym**
Right now we have to many of them and the way to fix is not totally clear. I'll file an issue to investigate and re-enable.

Also https://github.com/Microsoft/msbuild/blob/6851538897f5d7b08024a6d8435bc44be5869e53/src/Build/Evaluation/Evaluator.cs#L2498 indicates that MSBuild guys want to add some special treatment regarding this import and Sdk attributes.

### CODE REVIEW

**Review by pranavkm — APPROVED** [review]

File: build/KoreBuild.ps1 [hunk 1]
```diff
@@ -117,6 +117,7 @@ if ("${env:CI}${env:APPVEYOR}${env:TEAMCITY_VERSION}${env:TRAVIS}" -eq "")
```
Comment (pranavkm) [inline build/KoreBuild.ps1:117]: Don't check in the zip file

**Comment by pranavkm** [comment]
Can you edit the commit to say this is suppressing the duplicate import warning?

**Comment by natemcmaster** [comment]
We shouldn't import files that have an Sdk attribute.

**Comment by natemcmaster** [comment]
https://github.com/aspnet/KoreBuild/issues/216#issuecomment-295983850

**Comment by pakrym** [comment]
@natemcmaster I'll merge it to make it easier to diagnose Universe/Gate builds. We should the right way when we have time or doing changes in the area.

**Pull request merged**